var/spack/repos/builtin/packages/py-scikit-build-core/package.py

```diff
@@ -42,7 +68,7 @@ class PyScikitBuildCore(PythonPackage):
     # Test dependencies
     depends_on("py-build +virtualenv", type="test")
     depends_on("py-cattrs@22.2:", type="test")
-    depends_on("py-importlib-metadata", when="^python@:3.7", type="test")
+    depends_on("py-importlib-metadata", type=("build", "run"), when="@0.3: ^python@:3.7")
```

Comment:
I also now notice this is under test dependencies, but it should be under full dependencies, or at least it should *also* be there. We want py-importlib-metadata to continue to be a test dependency for all previous version, but it should now be a regular dependency for the newer versions after 0.3.

Reply:
Should be done

Reply:
Python 3.7 is no longer supported anyway so it doesn't really matter.